src/ripple/app/tx/impl/Transactor.cpp

```diff
@@ -57,7 +57,7 @@ preflight1(PreflightContext const& ctx)
 {
     // This is inappropriate in preflight0, because only Change transactions
     // skip this function, and those do not allow an sfTicketSequence field.
-    if (ctx.tx.getSeqProxy().isTicket() &&
+    if (ctx.tx.isFieldPresent(sfTicketSequence) &&
```

Comment:
Isn't this transaction breaking? The old check looked at both the field and the type, The new code only looks at the field. I.e. if there was a `sfTicketSequence` field with a string, the old code let it through and the new code flags it as malformed.

Reply:
Answering my own question. I see that `TicketBatch` has not been released yet.